scratch/fgpu/run-fgpu-hbw.sh

```diff
@@ -21,8 +21,9 @@ export CUDA_VISIBLE_DEVICES="$cuda1"
 
 set -x
 exec spead2_net_raw taskset -c $other_affinity fgpu \
-    --src-chunk-samples 67108864 \
+    --src-chunk-samples 134217728 \
     --dst-chunk-jones 33554432 \
+    --src-buffer=268435456 \
```

Comment:
It's a bit late for me to be asking what the significance of these numbers are (are they special in some way? Some multiple/power?).

I'll approve so long. Feel free to reply here or Mattermost or chat during the check-in.

Reply:
They're powers of 2.

Reply:
My brain wasn't up to temperature yet. I naively did `math.sqrt` on the new values, instead of `math.log2`. Of course they're powers of 2 🤦